test/test_torch.py

```diff
@@ -5483,6 +5483,98 @@ def test_dot_data_use(self):
             model.weight.data = weight
             out = model(input)
 
+    def test_bucketization(self):
```

Comment:
Please add the tests to device-generic framework (e.g. TestTorchDeviceType), it will iterate over device types for you. https://github.com/pytorch/pytorch/wiki/Writing-tests-that-run-on-all-available-device-types

Reply:
done